exonum-java-binding-common/pom.xml

```diff
@@ -55,10 +57,31 @@
     <dependency>
       <groupId>com.google.auto.value</groupId>
       <artifactId>auto-value</artifactId>
-      <version>1.6.2</version>
+      <version>${auto-value.version}</version>
       <scope>provided</scope>
     </dependency>
 
+    <dependency>
+      <groupId>com.ryanharter.auto.value</groupId>
+      <artifactId>auto-value-gson-annotations</artifactId>
+      <version>${auto-value-gson.version}</version>
+      <scope>compile</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>com.ryanharter.auto.value</groupId>
+      <artifactId>auto-value-gson</artifactId>
+      <version>${auto-value-gson.version}</version>
+      <scope>compile</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>com.squareup</groupId>
+      <artifactId>javapoet</artifactId>
+      <version>${javapoet.version}</version>
+      <scope>compile</scope>
```

Comment:
Is it used anywhere (no imports)?

Reply:
Transitive dependency for auto-value-gson, fails without it.